src/python/grpcio_testing/grpc_testing/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+package(default_visibility = ["//visibility:public"])
+
+py_library(
+    name = "common",
+    srcs = ["_common.py"],
+)
+
+py_library(
+    name = "time",
+    srcs = ["_time.py"],
+)
+
+py_library(
+    name = "grpcio_testing",
+    srcs = ["__init__.py"],
```

Comment:
I know creating a library for each file is more ideal for Bazel. But here we could use a glob to reduce future maintenance need. WDYT?

CC @gnossen

Reply:
Considering that this directory has not changed in years and we have no plans to change it in the future, I think this is okay for the moment.